spec.bs

```diff
@@ -420,13 +425,11 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
         1. If [=this=]'s [=module map=] is not [=map/empty=]:
             1. Let |operationMap| be [=this=]'s {{SharedStorageWorkletGlobalScope}}'s [=SharedStorageWorkletGlobalScope/operation map=].
             1. If |operationMap| [=map/contains=] |name|:
-                1.  Let |operation| be |operationMap|[|name|].
-                1. [=Assert=]: |operation|'s [=associated realm=] is [=this=]'s [=relevant realm=].
-                1. If |options| [=map/contains=] |data|:
-                    1. Let |argumentsList| be a new [=/list=].
-                    1. [=list/Append=] |data| to |argumentsList|.
-                    1. [=Invoke=] |operation| with |argumentsList|.
-                1. Otherwise, [=invoke=] |operation| without any arguments list.
+                1. [=Assert=]: |operationMap|[|name|]'s [=associated realm=] is [=this=]'s [=relevant realm=].
+                1. Let |operation| be |operationMap|[|name|], [=converted to an IDL value|converted=] to {{Function}}.
```

Comment:
Quick question (tangential to the CL but related): In the code, we define both [RunFunctionForSharedStorageRunOperation](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/shared_storage/shared_storage_worklet_global_scope.idl?q=RunFunctionForSharedStorageRunOperation) and [RunFunctionForSharedStorageSelectURLOperation](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/shared_storage/shared_storage_worklet_global_scope.idl?q=RunFunctionForSharedStorageSelectURLOperation). But in the spec, only `RunFunctionForSharedStorageSelectURLOperation` is defined. This is because for `RunFunctionForSharedStorageRunOperation`, the end behavior seems the same as calling the operation as a `{{Function}}`. I'm wondering if we should strive for consistency here.

Reply:
Since nothing actually waits on the promise returned from the operation (intentionally, according to the note), I suspect the difference is not web-observable. It would probably be good to make the two consistent. I'd rather not block this PR on that, but all else equal, easy correspondence between implementations and spec makes it easier to evolve the two in parallel.